proto/meta.proto

```diff
@@ -492,16 +499,17 @@ message GetClusterInfoResponse {
   uint64 revision = 5;
 }
 
-message Reschedule {
-  repeated uint32 added_parallel_units = 1;
-  repeated uint32 removed_parallel_units = 2;
+message WorkerReschedule {
+  map<uint32, uint32> increased_actor_count = 1;
+  map<uint32, uint32> decreased_actor_count = 2;
 }
 
 message RescheduleRequest {
-  // reschedule plan for each fragment
-  map<uint32, Reschedule> reschedules = 1;
+  reserved "reschedules";
+  reserved 1;
   uint64 revision = 2;
   bool resolve_no_shuffle_upstream = 3;
+  map<uint32, WorkerReschedule> worker_reschedules = 4;
```

Comment:
It might be worth commenting the meaning of each `uint32` here. Is it `fragment_id -> worker_id -> [+|-]cnt`?